src/iosrv/init.c

```diff
@@ -461,30 +463,39 @@ abt_fini(void)
 static int
 server_init(int argc, char *argv[])
 {
-	uint64_t	bound;
-	int64_t		diff;
-	unsigned int	ctx_nr;
-	char		hostname[256] = { 0 };
-	int		rc;
+	static struct d_tm_node_t	*startup_dur;
```

Comment:
Might be worthwhile to add a metric for the rank as well. If we're looking at an aggregated tree of metrics it would be useful to see the ranks in it. Could also be a way to query for rank via metrics rather than scraping logs or whatever.

Reply:
Done